### ISSUE TITLE
DOC: clarify that .ix[] is always label based with integer index

### ISSUE THREAD

**Issue by tianchuanting** [pydata/pandas#7663]
According to the documentation, ix should accept the all the input to iloc. In the following the .ix raises  an KeyError while .iloc work as expected. I suspect the fall back mechanism is broken in this case with integer index. 

```python
import pandas as pd
import numpy as np

df = pd.Series(np.random.randn(9))
df.iloc[-1]
df.ix[-1] # raise KeyError
  ```
version: 
pandas: 0.13.1
python: 2.7.6.final.0

EDIT: Changed title from "df.ix[-1] broken" to "DOC: clarify that .ix[] is always label based with integer index"

**Comment by clham** [comment]
This appears fixed as of 6aea232a9e05df7f478ba9a478e482442153fe80?

**Comment by jorisvandenbossche** [comment]
yes, indeed that seems so. Thanks for noticing!

**Comment by jorisvandenbossche** [comment]
Closed by  #8262
